### DISCUSSION TITLE
Some websites don't seem to show in the activity tab

### DISCUSSION THREAD

**Discussion by isle9** [ActivityWatch/activitywatch#1014] [Q&A]
Does anyone know why some websites don't seem to show up in the activity tab, even though they appear in the timeline?

![image](https://github.com/ActivityWatch/activitywatch/assets/74906541/22804b08-d0cb-49fe-9d2d-5ebfc01d696e)
![image](https://github.com/ActivityWatch/activitywatch/assets/74906541/8e25145c-f17d-4226-9318-a69efda1dd1d)
![image](https://github.com/ActivityWatch/activitywatch/assets/74906541/c0b06365-052d-4e58-975b-70e69e8c4a4c)

Twitch is in green in the timeline, but in activity it's empty. It doesn't show anywhere else either (in the activity tab).
Disabling exclude AFK time doesn't fix it either.

**Comment by isle9** [comment]
Seems to be a Wayland issue. Didn't even know it wasn't supported and I used it for months now already...

**Comment by koljapluemer** [comment]
Did you have Twitch open/focused while not being afk? 

I always thought that afk time (and time spent on other windows while technically visiting the url in the hidden browser window) is excluded from the actual minute overview.

But I'm realizing now that this is just a guess, I don't actually know whether this is what's happening.

**Reply by isle9** [reply]
Yes, Twitch was open the whole time, and yeah, I don't think it tracks windows in the background at all.
This was definitely connected with me using Wayland since it worked before I switched to it, and it works again after I switched back.
